metricbeat/module/vsphere/host/host.go

```diff
@@ -62,7 +62,7 @@ func New(base mb.BaseMetricSet) (mb.MetricSet, error) {
 func (m *MetricSet) Fetch() ([]common.MapStr, error) {
 	f := find.NewFinder(m.Client, true)
 	if f == nil {
```

Comment:
Unfortunately `find.NewFinder` is lacking godocs to document the function's contract. But based on the implementation, the function cannot return `nil` so this check is unnecessary.